### DISCUSSION TITLE
NODDI model dipy

### DISCUSSION THREAD

**Discussion by rosella1234** [dipy/dipy#3046] [Reconstruction] [answered]
Hello,
I would like to know whether the NODDI model for multi-shell reconstruction is implemented in dipy library.
Indeed, I would like to apply a command equivalent to the DKI model 
```
dki_model = dki.DiffusionKurtosisModel(gtab)
    dki_slice = xval.kfold_xval(dki_model, data_slice, 2)
```
thanks in advance
Rosella

**Comment by arokem** [comment, marked as answer]
Hello! DIPY does not have an implementation of the NODDI model, but you might want to check out [AMICO](https://github.com/daducci/AMICO)

**Comment by rosella1234** [comment]
Thank you!
I have actually already computed NODDI maps through amico .
Just wanted to know whether corresponding commands I have posted for model
comparison for DKI data can be done for NODDI too . And in case how.
Thank you
Rosella

**Comment by rosella1234** [comment]
Sorry, is it then possible to reproduce a command equivalent to 

```
dki_model = dki.DiffusionKurtosisModel(gtab)
dki_slice = xval.kfold_xval(dki_model, data_slice, 2)
```
in case of NODDI model computed with AMICO ? 
Specifically, I don't know how to replace _DiffusionKurtosisModel_ in this case

Thanks in advance,
Rosella

**Reply by arokem** [reply]
No, unfortunately the API for AMICO is quite different from DIPY's model API, so that will not be straightforward to do. It should be possible, but will require some extra work. But it is a really great idea to compare DKI and AMICO, so I hope that you figure out how to do that with these two different software libraries!
